AbletonMCP_Remote_Script/__init__.py

```
@@ -1,1062 +1,1068 @@
+            self.server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            self.server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+            self.server.bind((HOST, DEFAULT_PORT))
+            self.server.listen(5)
+
+            self.running = True
+            self.server_thread = threading.Thread(target=self._server_thread)
+            self.server_thread.daemon = True
+            self.server_thread.start()
+
+            self.log_message("Server started on port " + str(DEFAULT_PORT))
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)